### ISSUE TITLE
BasicLTIConstants class CONTEXT_TYPE_GROUP is having a wrong value "GROUP"

### ISSUE THREAD

**Issue by anurag-thakre** [IMSGlobal/basiclti-util-java#23]
Hi All,

I found one issue while using BasicLTIConstants.CONTEXT_TYPE_GROUP constant variable.
Issue is, as per the LTI specification (https://www.imsglobal.org/specs/ltiv1p1/implementation-guide) ContextType should have following 
CourseTemplate,
CourseOffering,
CourseSection,
Group

All constants values are mentioned proper in BasicLTIConstants.java class but for CONTEXT_TYPE_GROUP it as follows 

public static final String CONTEXT_TYPE_GROUP = "GROUP";

So basically it should be "Group"

https://github.com/IMSGlobal/basiclti-util-java/blob/master/src/main/java/org/imsglobal/lti/BasicLTIConstants.java

Thanks,
Anurag

**Comment by pfgray** [comment]
Thanks!

**Comment by anurag-thakre** [comment]
Thanks Pfgray.
